README.rst

```diff
@@ -105,6 +111,7 @@ supports the following events:
 - ``subscribe``: Subscribe to a topic
 - ``message``: Send a message to a topic
 - ``unsubscribe``: Unsubscribe from a topic
+- ``ping``: Maintain and monitor connectivity
```

Comment:
I'd just say "monitor".  The connectivity is technically maintained at the protocol level.

Reply:
Done in 6707e95.